pages/settings.tsx

```diff
@@ -38,7 +38,8 @@ const Settings = () => {
               <h1 className="text-xs-center">{title}</h1>
               <SettingsForm />
               <hr />
-              <button className="btn btn-outline-danger" onClick={handleLogout}>
+              <button className="btn btn-outline-danger" onClick={handleLogout}
+              data-cy="settings-logout-btn">
```

Comment:
It's recommended to place the 'data-cy' attribute on a separate line for better readability in the code.

Reply:
fixed